### PR TITLE
AZP: Add support ubuntu 22.xx

### PR DESCRIPTION
**Pull request by avildema**
Add support Ubuntu 22.04 and 22.10

### CODE REVIEW

File: buildlib/pr/main.yml [hunk 1]
```diff
@@ -43,6 +43,12 @@ resources:
     - container: ubuntu2004
       image: rdmz-harbor.rdmz.labs.mlnx/swx-infra/x86_64/ubuntu20.04/builder:mofed-5.0-1.0.0.0
       options: $(DOCKER_OPT_VOLUMES)
+    - container: ubuntu2204
+      image: rdmz-harbor.rdmz.labs.mlnx/swx-infra/x86_64/ubuntu20.10/builder:mofed-5.7-0.2.3.0
+      options: $(DOCKER_OPT_VOLUMES)
+    - container: ubuntu2210
+      image: rdmz-harbor.rdmz.labs.mlnx/swx-infra/x86_64/ubuntu20.10/builder:mofed-5.8-0.2.1.0
```
Comment (yosefe) [inline buildlib/pr/main.yml:50]: is this the lowest MLNX_OFED version that supports Ubuntu 22.10?

**Review by yosefe — COMMENTED** [review]
LGTM in general, may need to push fixes if build fails
Thanks @avildema !

File: buildlib/pr/main.yml (hunk 1 above)
Comment (avildema) [reply buildlib/pr/main.yml:50]: yes, I checked

**Comment by yosefe** [comment]
@brminich can you pls push fix to this PR as part of https://redmine.mellanox.com/issues/3205556 (Internal link) ?

**Comment by brminich** [comment]
@avildema, plz enable `edit by maintainers` check-box

**Review by hoopoepg — APPROVED** [review]

**Review by yosefe — APPROVED** [review]

**Pull request merged**